k8s/apis/workflow/v1/activation_types.go

```diff
@@ -17,10 +17,9 @@ import (
 type ActivationStatus struct {
 	Status               v1alpha2.State `json:"status,omitempty"`
 	StatusMessage        string         `json:"statusMessage,omitempty"`
-	ErrorMessage         string         `json:"errorMessage,omitempty"`
 	ActivationGeneration string         `json:"activationGeneration,omitempty"`
 	UpdateTime           string         `json:"updateTime,omitempty"`
-	StageHistory         []StageStatus  `json:"stagehistory,omitempty"`
+	StageHistory         []StageStatus  `json:"stageHistory,omitempty"`
```

Comment:
Please inform @FireDefend about this change.

Reply:
Yes. Shawn already knew that, and the change is in the swagger change,